package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "tracker-radar",
+  "version": "1.0.0",
+  "description": "Data set of the most common third party domains on the web with information about their behavior, classification and ownership.",
+  "scripts": {
+    "test": "mocha --timeout 30000"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/duckduckgo/tracker-radar.git"
+  },
+  "author": "DuckDuckGo",
+  "license": "CC-BY-NC-SA-4.0",
```

Comment:
Our license file says "CCBY-NC-SA 4.0" no idea if both of those are considered valid.

Reply:
Not sure, changed it so they match at least.